### PR TITLE
Add a function to tidy LymphGen Subtype.Prediction columns

### PR DESCRIPTION
**Pull request by lkhilton**

### CODE REVIEW

**Comment by Kdreval** [comment]
@lkhilton , @mattssca , or @rdmorin Please review.
This fixes the bug of COMPOSITE in lymphgen metadata, adds a new function to collate lymphgen results, and also implements catchall for pathologies that are under review.
Thanks!

**Review by mattssca — APPROVED** [review]
Good updates, I tested the added functions and they do what's advertised. Just a minor comment on the colour for "OTHER". Thanks!

File: R/utilities.R [hunk 1]
```diff
@@ -1681,6 +1681,7 @@ get_gambl_colours = function(classification="all",alpha=1){
       "MM"="#CC9A42",
       "SCBC"="#8c9c90",
       "UNSPECIFIED"="#cfba7c",
+      "OTHER"="#cfba7c",
```
[the remark this answers is not in the document]
Comment (Kdreval) [reply R/utilities.R:1684]: Thanks Adam! Yes, light gray and the same color as Unspecified on purpose.

[the remark this answers is not in the document]
Comment (mattssca) [reply R/utilities.R:1684]: Great, just wanted to double-check!

**Pull request merged**